### DISCUSSION TITLE
How To Get Brand Colors Into  HCT and Color Scheme has changed from M2

### DISCUSSION THREAD

**Discussion by fredgrott** [rydmike/flex_color_scheme#251] [Q&A] [answered]
Okay, 

I just realize this upon examining code....

CorePalette android side in MCU does not have PrimaryDim and PrimaryFixed or their on colors.

But DynamicScheme has them....

So its now copywith PrimaryDim PrimayFixed Error and ErrorContainer and their on colors with the MCU Blend.harmonize method using our Primary Brand color

That way it works with the static ColorScheme variant and with dynamic color out of box. The Blend.harmonize method will shift the hue color of the generated color towards the Brand primary hue.

That also directly implies that we should start incorporating used of PrimaryFixed and PrimaryDim in our app designs.

**Comment by rydmike** [comment, marked as answer]
By the way and for info FCS 8.0.0 is out as early access as 8.0.0-dev.1

It has all the new colors in ColorScheme and can also use Flutter's and MCU's DynamicSchemeVariant, even with multiple seed colors, even MCU does not support that 😄 

Also with both flex_seed_scheme and flex_color_scheme getting brand colors into them works as before. 


**FlexColorScheme** package dev release 8.0.0-dev.1 has been published. This is a BIG update with breaking changes and many new features. It has its own dev pre-release of the [**Themes Playground**](https://rydmike.com/flexcolorscheme/themesplayground-dev) as well, feel free to take a look. No new API changes are planned for the stable 8.0.0 release from this dev.1 release.

We would appreciate any feedback about found issues from early adopters. 

* [Doc site has some pre-info](https://docs.flexcolorscheme.com/)
* [Dev release of PKG here](https://pub.dev/packages/flex_color_scheme/versions/8.0.0-dev.1)
* [The PKG changelog has the most detailed info](https://pub.dev/packages/flex_color_scheme/versions/8.0.0-dev.1/changelog)

**FOR INFO** there will be no new features or changes in it compared to this one. Only any issues found from the quality assurance will be addressed. The stable release will also include many doc updates, but docs site will continue to be edited and revised AFTER the 8.0.0 stable release.

**Reply by rydmike** [reply]
And now it is released as 8.0.0 stable

- https://github.com/rydmike/flex_color_scheme
- https://rydmike.com/flexcolorscheme/themesplayground-latest/
- https://docs.flexcolorscheme.com/whats_new
